doc/GUIDE.md

```diff
@@ -1939,213 +1939,8 @@ can use the `stack dot` command and Graphviz. More information is
 
 ### Travis with caching
 
-Many people use Travis CI to test out a project for every Git push. We have [a
```

Comment:
Commit pushed. I couldn't follow the link you provided, but I think I found the only two references in the repo.